### DISCUSSION TITLE
Eliminar Pagina por completo

### DISCUSSION THREAD

**Discussion by ILoveYouF** [community/community#148004] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Quiero eliminar mi pagina web publicada desde git hub por completo, ya elimine el repositorio pero aun sigue apareciendo en internet y no quiero eso.

**Comment by jgcamiloaga** [comment]
Eliminar el sitio de GitHub Pages:

1. Accede a tu cuenta de GitHub y abre la sección Settings del repositorio que utilizaste para GitHub Pages.
2. Dentro de la configuración, busca la sección GitHub Pages.
3. En la sección GitHub Pages, junto a "Visit site", aparece la opción para deshabilitar el sitio web.
De esta manera, el sitio dejará de estar disponible en línea.

![Captura de pantalla 2024-12-27 112921](https://github.com/user-attachments/assets/961917e8-a565-4d35-a4df-279df19f03e0)

**Reply by ILoveYouF** [reply]
Pasa que ya elimine el repositorio :C

**Reply by jgcamiloaga** [reply]
Creo que se puede solucionar borrando la caché del navegador, porque en realidad la página ya no existe.

**Reply by ILoveYouF** [reply]
y como haria eso?

**Reply by jgcamiloaga** [reply]
> y como haria eso? 
> 

La página web que hiciste se visualiza desde otros dispositivos? O solo desde el dispositivo que subiste a Internet?

**Reply by ILoveYouF** [reply]
desde otros, esta publico en internet, ya no aparece nada cuando ingresas al link pero aun sigue apareciendo el nombre y la foto de la pagina

**Comment by mecodeatlas** [comment]
Hi @ILoveYouF, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
